sysconfigs/daint/packages.yaml

```diff
@@ -86,6 +90,12 @@ packages:
     gawk:
         paths:
              gawk@4.2.1: /usr
+
+    gcc:
+        buildable: false
+        modules:
+          gcc@8.1.0: PrgEnv-gnu gcc/8.1.0
+          gcc@9.3.0: PrgEnv-gnu gcc/9.3.0
```

Comment:
Are you 100% sure this works? I tried several time and it didn't.

Reply:
It works, I checked.